rero_ils/modules/notifications/api.py

```diff
@@ -65,6 +65,22 @@ class Meta:
 
         default_filter = None
 
+    def get_claims(self, item_pid):
+        """Get the claims notifications about an issue item.
+
+        :param item_pid: the item pid related to the claim notification.
+        :returns: the list of claim notification hit from ES.
+        :rtype: list<Hit>.
+        """
+        # DEV NOTES :: Why returning a list instead of generator.
+        #   We don't use `yield` notation to avoid "Timeout" ElasticSearch
+        #   exception. Additionally, the number of claims notification for the
+        #   same issue item shouldn't be a large set of hit.
+        query = self \
+            .filter('term', context__item__pid=item_pid) \
+            .filter('term', notification_type=NotificationType.CLAIM_ISSUE)
+        return list(query.scan())
```

Comment:
A general comment here: is it a good idea to force to return a list? By doing this we do not let the possibility to load a large set of values using an iterator.